### ISSUE TITLE
Insert mode mapping for copy paste

### ISSUE THREAD

**Issue by HaleTom** [akahuku/wasavi#170]
I'd really like to have Ctrl-V work as per usual.

Having to press:
1. `Escape`
2. `"*p`
3. `i`

is tedious.

How do I create an insert mode mapping for Ctrl-C/V/X?

**Comment by HaleTom** [comment]
I tried to use:
```
map! <C-V> <esc>"*pa
map! <C-C> "*y
map! <C-X> "*d
```

But pasting inserts a newline, the pasted text, then another newline... Hmmm.

**Comment by akahuku** [comment]
wasavi can paste register contents by pressing `Ctrl+R` `<register name>` in input mode.
So `Ctrl+R` `*` pastes clipboard contents.
It is the same as vim!

**Comment by butla** [comment]
@akahuku You're correct, although what's also possible in normal Vim is to use the system clipboard as the default register, which gives you yank (`y`) and paste (`p`) "integrated" with the desktop environment, which in turn is very nice :) And I use that, but I'm aware that some people don't prefer it.

So would putting `set clipboard=unnamedplus` in wasavi do anything? Is that feature implemented?

Cause it would make this plugin (I'm just starting to use it) be very cool. But even now it's cool. Hell, I'm writing this note in it :D Together with Vimium they're almost perfect.

**Comment by andys8** [comment]
Ist there a known way to get `<C-v>` or `<C-V>` to work?

**Comment by RGD2** [comment]
You can now just use `map! <c-v> <c-r>*`

**Comment by RGD2** [comment]
Although I don't recommend it - it means you won't be able to insert utf-8 characters with `<c-v>u`, since you'll cover over the special insert mode.

**Comment by mmikowski** [comment]
RGD2: I mapped to <c-V> (ctrl + shift + v) and that worked great and doesn't interfere with the native <c-v> behavior.
